### PR TITLE
Datatables language

### PR DESCRIPTION
**Pull request by HB9HIL**

### CODE REVIEW

File: application/views/interface_assets/footer.php [hunk 1]
```diff
@@ -54,7 +54,17 @@
...
         $('[data-bs-toggle="tooltip"]').tooltip();
     });
-    </script>
+</script>
+
+<script>
+    function get_datatables_language() {
+        lang_datatables_language = "<?php echo lang('datatables_language'); ?>";
+        datatables_language_url = "../assets/json/datatables_languages/" + lang_datatables_language + ".json";
+        console.log(datatables_language_url);
```
Comment (int2001) [inline application/views/interface_assets/footer.php:63]: Remove dbg-output

**Review by int2001 — APPROVED** [review]
Working. Bug is away.
Remove dbg-output and merge?!

File: application/views/interface_assets/footer.php (hunk 1 above)
Comment (HB9HIL) [reply application/views/interface_assets/footer.php:63]: ah yes

**Comment by HB9HIL** [comment]
> Working. Bug is away. Remove dbg-output and merge?!

dbg-output is removed

and no, 
not merge ready yet. Still some datatables are missing and not working properly. 

we use the Datatable JS around 70 times in the complete Code

**Comment by int2001** [comment]
assign review when finished. i'll test all datatables afterwards. Tested only LBA so far

**Review by HB9HIL — COMMENTED** [review]
I think this one is ready 2 merge. I compared with Tamh's Code. Is is just using another name for the function and using a faulty code in footer. My version is working properly as I can see that. 

Important to say, that the datatable languages in the public view's like public log and oqrs are hardcoded to english as this is international.

**Review by int2001 — CHANGES_REQUESTED** [review]
working tested:
- Bandlist (DXCluster)
- Bands
- Anonymous Slug OQRS
- Some random Award-Pages

Not working:
- oqrs.js (Throwing JS: `jquery-3.3.1.min.js:2  GET https://l[url]/index.php/assets/json/datatables_languages/en-GB.json 404 (Not Found)` - invoked by L74
- Anonymous Slug Dashboard - function `getDataTablesLanguageUrl` not found
- Anonymous Slug Gridsquares - function `getDataTablesLanguageUrl` not found

**Comment by HB9HIL** [comment]
> working tested:
> 
> * Bandlist (DXCluster)
> * Bands
> * Anonymous Slug OQRS
> * Some random Award-Pages
> 
> Not working:
> 
> * oqrs.js (Throwing JS: `jquery-3.3.1.min.js:2  GET https://l[url]/index.php/assets/json/datatables_languages/en-GB.json 404 (Not Found)` - invoked by L74
> * Anonymous Slug Dashboard - function `getDataTablesLanguageUrl` not found
> * Anonymous Slug Gridsquares - function `getDataTablesLanguageUrl` not found

I disabled the multilanguage for datatables in the public views, as the default is english and as we do not have a selectable language in the public views now it should stay english.

**Pull request merged**